test/validatorCompiler.js

```diff
@@ -0,0 +1,126 @@
+const {
```

Comment:
add use strict at the top

Reply:
I added all as you asked

Reply:
please let me know If I left anything

Reply:
he means `"use strict";` for enable the strict mode.

Reply:
yes, but I think he is looking for anything else too

Reply:
I will add this tomorrow morning